lua/astrocommunity/completion/tabby-nvim/README.md

```diff
@@ -0,0 +1,55 @@
+# Mac/Windows Installation
+
+https://tabby.tabbyml.com/docs/installation/
```

Comment:
This is more than enough. People should look at the official documentation. Everything below this can be removed.

Reply:
Cool, will just change it to a more general "Installation" then